src/main/java/sirius/biz/tenants/UserAccountData.java

```diff
@@ -86,6 +86,11 @@
     @AutoImport
     private boolean externalLoginRequired = false;
 
+    public static final Mapping LANG = Mapping.named("lang");
+    @Autoloaded
+    @NullAllowed
```

Comment:
alternative we could set the `lang` in `@BeforeSave` to the `NLS.getDefaultLanguage()` if empty